queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -184,10 +184,35 @@ def reduction_started(self):
                      self.message.serialize(limit_reduction_script=True))
         self.client.send(ACTIVEMQ_SETTINGS.reduction_started, self.message)
 
+    def result_and_log_directory(self, temporary_root_directory, reduce_dir):
```

Comment:
Could this be named to something more descriptive?

I prefer something like `create_result_and_log_directories` but I am not fussed. 

I think a string of nouns is always confusing for a func/method name, and I think the singular directory is misleading, when a tuple is returned.